doc/rst/technotes/initializers.rst

```diff
@@ -740,7 +740,7 @@ types use initializers when possible.  Please report any bugs
 encountered using the guidance described at the `bugs`_ page.
 
 .. _bugs:
-   https://chapel-lang.org/docs/latest/usingchapel/bugs.html
+   https://chapel-lang.org/docs/usingchapel/bugs.html
```

Comment:
In this case, I think it's OK to stick to the latest version, since we'll always want users to get the latest bugs page